.github/ct.yaml

```diff
@@ -2,6 +2,8 @@ remote: origin
 target-branch: main
 chart-dirs:
   - charts
+excluded-charts:
+  - cloudprem
```

Comment:
`ct` (chart-testing) is use to detect which chart is modified by a pull request.
If the `cloudprem` chart is excluded the CI will return an error saying that the PR doesnt modify any chart. In that case you will need to add the label `no-chart`

Reply:
yep, found that out the hard way